clients/client-py/taskcluster/helper.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+import logging
+import json
+from taskcluster.generated import _client_importer
+
+logger = logging.getLogger(__name__)
+
+
+class TaskclusterConfig(object):
+    """
+    Local configuration used to access Taskcluster service and objects
+    """
+
+    def __init__(self, url="https://community-tc.services.mozilla.com"):
```

Comment:
We have avoided having any default rootUrls.  Callers should be explicit about the deployment they are using.

Reply:
Makes sense, i'll remove that.